.github/workflows/render.yml

```diff
@@ -1,25 +1,14 @@
-name: Build tex and rst
+name: Check Renderability
 
-on: 
-  workflow_dispatch:
-  push:
-    branches:
-    - main
+on:
+  pull_request:
 
 jobs:
-
   render:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4.1.7
```

Comment:
Add a check here that the `rendered/` folder does not exist before the CI render runs.